src/Control/Effect/Fail.hs

```diff
@@ -2,7 +2,7 @@
 
 {- | An effect providing failure with an error message.
 
-This effect is invoked through the 'Fail.fail' method from 'Fail.MonadFail'.
+This effect is invoked through the 'Control.Monad.Fail.fail' method from 'Control.Monad.Fail.MonadFail'.
```

Comment:
Wait a sec, this isn’t able to resolve via the named import? Well there goes my idea about importing just the type synonym in `Control.Carrier.Fail.Either` 😞